src/seedu/addressbook/data/person/Address.java

```diff
@@ -21,10 +21,12 @@
      * @throws IllegalValueException if given address string is invalid.
      */
     public Address(String address, boolean isPrivate) throws IllegalValueException {
```

Comment:
Avoid throwing exceptions from the constructor. It is not a good practice. The purpose of constructor should be clean and clear. Input validations should be done at a front level before calling the constructor.